google/cloud/bigtable/client_options_test.cc

```diff
@@ -307,8 +278,9 @@ TEST(ClientOptionsTest, SetGrpclbFallbackTimeoutMS) {
   bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
   ASSERT_STATUS_OK(client_options_object.SetGrpclbFallbackTimeout(
       std::chrono::milliseconds(5)));
-  auto const actual = GetInt(client_options_object.channel_arguments(),
-                             GRPC_ARG_GRPCLB_FALLBACK_TIMEOUT_MS);
+  auto const actual = ::google::cloud::internal::GetIntChannelArgument(
```

Comment:
Nit: Do we want to fully-qualify names like this?

Reply:
I guess not.